service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/disk/CreateAzureDiskStepTest.java

```diff
@@ -31,9 +31,7 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mock;
-import org.springframework.test.context.ActiveProfiles;
 
-@ActiveProfiles("azure")
 public class CreateAzureDiskStepTest extends BaseAzureTest {
```

Comment:
I _think_ all the Azure `*StepTest`s can be `BaseAzureUnitTest`. (happy to do that in a separate PR too)